apps/onnx/onnx_converter_test.cc

```diff
@@ -504,6 +510,4 @@ int main() {
     test_concat();
     test_constant_fill();
     test_model();
-    printf("Success!\n");
```

Comment:
Please restore these two lines (printing something on success is convention for Halide tests)